packages/vuetify/src/components/VToolbar/VToolbar.sass

```diff
@@ -0,0 +1,130 @@
+@import '../../styles/styles.sass'
+@import './_variables.scss'
+
+// Block
+.v-toolbar
+  transition: $toolbar-transition
+
+  +elevation(4)
+
+  .v-input
+    padding-top: 0
+    margin-top: 0
+
+// Element
+.v-toolbar__content,
+.v-toolbar__extension
+  padding-bottom: 4px
+  padding-top: 4px
+  padding-left: 16px
+  padding-right: 16px
+
+  > .v-btn.v-btn--icon
+    height: 48px
+    width: 48px
+
+    &:first-child
+      margin-left: -12px
+
+      + .v-toolbar__title
+        padding-left: 20px
+
+    &:last-child
+      margin-right: -12px
+
+    +rtl()
+      &:first-child
+        margin-right: -12px
+
+        + .v-toolbar__title
+          padding-right: 20px
+
+      &:last-child
+        margin-left: -12px
+
+  > .v-tabs
+    height: inherit
+    margin-top: -4px
+    margin-bottom: -4px
+
+    .v-tabs-bar
+      height: inherit
+
+    &:first-child
+      margin-left: -16px
+
+    &:last-child
+      margin-right: -16px
+
+.v-toolbar__content,
+.v-toolbar__extension
+  align-items: center
+  display: flex
+
+.v-toolbar__image
+  border-radius: inherit
+  position: absolute
+  top: 0
+  bottom: 0
+  width: 100%
+  z-index: -1
+  contain: strict
+
+  .v-image
+    border-radius: inherit
+
+.v-toolbar__items
+  display: flex
+  height: inherit
+
+  > .v-btn
+    border-radius: 0
+    height: 100% !important
+    max-height: none
+
+.v-toolbar__title
+  font-size: 1.25rem
+  overflow: hidden
+  text-overflow: ellipsis
+  white-space: nowrap
+
+// Modifier
+.v-toolbar.v-toolbar--absolute
+  position: absolute
+  top: 0
+  width: 100%
+  z-index: 1
+
+.v-toolbar.v-toolbar--bottom
+  top: initial
+  bottom: 0
+
+.v-toolbar.v-toolbar--collapse
+  .v-toolbar__title
+    white-space: nowrap
+
+.v-toolbar.v-toolbar--collapsed
+  border-bottom-right-radius: 24px
+  width: 112px
```

Comment:
maybe var it?

Reply:
or prop it?

Reply:
Maybe in the future. As it stands, it is setup to explicitly support 2 icons when collapsed and hides the rest.